try{
	throw new Error('my silly error');
} catch(error) {
	console.log(error);
}